src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx

```diff
@@ -1,33 +1,132 @@
-          <div>
-            <div>{skip + index + 1}</div>
-          </div>
-          <div>
-            {member.firstName} {member.lastName}
+export const ProjectMemberTable = ({ projectMembers, skip, take, handleMemberCount }) => {
+  const [allMemberList, setAllMemberList] = useState([]);
+  const [activeMemberList, setActiveMemberList] = useState([]);
+  const [memberFilter, setMemberFilter] = useState('active');
+  const { fetched, foundUsers, members } = projectMembers;
+
+  let activeMemberTable = [];
+  let allMemberTable = [];
+
+  useEffect(() => {
+    let memberList = [];
+    let activeList = [];
+    let currentActive = [];
+    if (fetched) {
+      if (foundUsers.length > 0) {
+        foundUsers.map(member => {
+          currentActive.push(member._id);
+        });
+      }
+      members.map(member => {
+        if (currentActive.includes(member._id)) {
+          memberList.push({ ...member, active: true });
+          activeList.push({ ...member, active: true });
+        } else {
+          memberList.push({ ...member, active: false });
+        }
+      });
+      setAllMemberList(memberList);
+      setActiveMemberList(activeList);
+    }
+  }, [fetched]);
+
+  activeMemberTable = activeMemberList.slice(skip, skip + take).map((member, index) => (
+    <div className="project-member-table-row" id={'tr_' + member._id} key={'ac_' + member._id}>
+      <div>
+        <div>{skip + index + 1}</div>
+      </div>
+      <Link to={`/userprofile/${member._id}`} title="View Profile">
+        <div>
+          {member.firstName} {member.lastName}
+        </div>
+      </Link>
+      <div className="projects__active--input">
+        {member.active ? (
+          <tasks className="isActive">
+            <i className="fa fa-circle" aria-hidden="true"></i>
+          </tasks>
+        ) : (
+          <div className="isNotActive">
+            <i className="fa fa-circle-o" aria-hidden="true"></i>
           </div>
-          <div>{member._id}</div>
+        )}
+      </div>
+      <div>{member._id}</div>
+    </div>
+  ));
+
+  allMemberTable = allMemberList.slice(skip, skip + take).map((member, index) => (
+    <div className="project-member-table-row" id={'tr_' + member._id} key={'al_' + member._id}>
+      <div>
+        <div>{skip + index + 1}</div>
+      </div>
+      <Link to={`/userprofile/${member._id}`} title="View Profile">
+        <div>
+          {member.firstName} {member.lastName}
         </div>
-      ));
-    }
-  }
+      </Link>
+      <div className="projects__active--input">
+        {member.active ? (
+          <tasks className="isActive">
+            <i className="fa fa-circle" aria-hidden="true"></i>
+          </tasks>
+        ) : (
+          <div className="isNotActive">
+            <i className="fa fa-circle-o" aria-hidden="true"></i>
+          </div>
+        )}
+      </div>
+
+      <div>{member._id}</div>
+    </div>
+  ));
```

Comment:
Could `activeMemberTable` and `allMemberTable` be declared here as `const` rather than as `let` on lines 12 and 13?